README.md

```diff
@@ -1,23 +1,42 @@
-## argocd-trivy-extension
+# argocd-trivy-extension
+
 Argo CD UI extension that displays vulnerability report data from [Trivy](https://aquasecurity.github.io/trivy), an open source security scanner.
 
 `Trivy` creates a vulnerability report Kubernetes resource with the results of a security scan. The UI extension then parses the report data and displays it as a grid and dashboard viewable in Pod resources within the Argo CD UI.
 
-<img src="./docs/table.png" width="85%" display="inline-flex"/> <img alt="vulnerabilities dashboard" src="./docs/dashboard.png" width="85%"/>
```

Comment:
This is cosmetic but I prefer the image sized to 85%. I feel it makes the document more readable. Debatable, but can we revert just this line.

Reply:
fixed